modules/openxr/register_types.cpp

```diff
@@ -115,6 +116,7 @@ void initialize_openxr_module(ModuleInitializationLevel p_level) {
 			OpenXRAPI::register_extension_wrapper(memnew(OpenXRDisplayRefreshRateExtension));
 			OpenXRAPI::register_extension_wrapper(memnew(OpenXRWMRControllerExtension));
 			OpenXRAPI::register_extension_wrapper(memnew(OpenXRML2ControllerExtension));
+			OpenXRAPI::register_extension_wrapper(memnew(OpenXRHandInteractionExtension));
```

Comment:
Move the property check here instead to avoid creating the extension if it's not used.

Reply:
Yup will do!

Reply:
Ok, one small drawback for this, but if we don't add the extension at all, we also do not register the meta data for the interaction profile. This means we must restart Godot after enabling this extension before the action map can be configured, and if we disable the extension, the profile will come up as unknown and can't be edited.

I think this is acceptable but we have to ensure this will be part of the manual pages once we write them.